requirements.txt

```diff
@@ -1,3 +1,4 @@
 virtualenv==16.7.10
 asv==0.4.1
 gitpython
+dvc==1.0.0a
```

Comment:
You will regret this :) We still break some stuff in alpha. Would use `0.94.0` unless you need something specific.